### PR TITLE
Adding deploy for sale app

### PR DESCRIPTION
**Pull request by jerrybassat7**
Deploy Script for testing the Sale app

### CODE REVIEW

File: test/NftFarm.test.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ describe("NftFarm", function () {
     const amount = 5;
     await wl.mintBatch(whitelisted.address, [id], [amount], []);
     await wl.setBurnerForID(nft.address, id);
-    await nft.setWhitelist(wl.address, (await getCurrentTimestamp()) + 1e4);
+    await nft.setWhitelist(wl.address, ( getCurrentTimestamp()) + 1e4);
```
Comment (sullof) [inline test/NftFarm.test.js:38]: Why this change? Without the await, `getCurrentTimestamp()` will return a promise.

Comment (jerrybassat7) [reply test/NftFarm.test.js:38]: VS code was telling it had no effect, But i will add it back.

Comment (sullof) [reply test/NftFarm.test.js]: Actually, you are right.
Debugging it, I see that that is not `getTimestamp` from the helpers, but `getCurrentTimestamp` from Hardhat, which is a sync function. I will reverse the change.
